website/docs/core-library/h3indexing.md

```diff
@@ -22,9 +22,9 @@ An `H3Index` is the integer representation of an H3 index, which may be one of m
 
 * Mode 0 is reserved and indicates an invalid H3 index.
 * Mode 1 is an *H3 Cell* (Hexagon/Pentagon) index.
-* Mode 2 is an *H3 Unidirectional Edge* (Cell A -> Cell B) index.
-* Mode 3 is planned to be a bidirectional edge (Cell A <-> Cell B).
-* Mode 4 is an *H3 Vertex* (i.e. a single vertex of an H3 Cell).
+* Mode 2 is an *H3 Directed Edge* (Cell A -> Cell B) index.
+* Mode 3 is an *H3 Nondirected Edge* (Cell A <-> Cell B) index.
```

Comment:
Nit: As above, I'd use "undirected" instead, here and below:
```suggestion
* Mode 3 is an *H3 Undirected Edge* (Cell A <-> Cell B) index.
```

Reply:
Or `Mode 3 is an *H3 (Undirected) Edge*` to highlight that, while it is undirected, we'll usually just refer to it as an "edge".